sdk/data/azcosmos/cosmos_container.go

```diff
@@ -486,8 +486,20 @@ func (c *ContainerClient) DeleteItem(
 
 // NewQueryItemsPager executes a single partition query in a Cosmos container.
 // query - The SQL query to execute.
-// partitionKey - The partition key to scope the query on.
+// partitionKey - The partition key to scope the query on. See below for more information on cross partition queries.
 // o - Options for the operation.
+//
+// Limited cross partition queries ARE possible with the Go SDK.
+// You can specify an empty list of partition keys by passing `NewPartitionKey()` to the `partitionKey` parameter, to indicate that the query WHERE clauses will specify which partitions to query.
```

Comment:
nit: How about swap below two lines, that way all in-side query context will be co-located.